<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PMan</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/std/icons/icons8-open-box-48.png</normaloff>:/std/icons/icons8-open-box-48.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow::separator {
    width: 5px;
    border-right:1px solid #AAAAAA;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageProjects">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QMdiArea" name="mdiAreaProjects">
          <property name="viewMode">
           <enum>QMdiArea::TabbedView</enum>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="tabsMovable">
           <bool>true</bool>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::South</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageJournals">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QMdiArea" name="mdiAreaJournals">
          <property name="viewMode">
           <enum>QMdiArea::TabbedView</enum>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="tabsMovable">
           <bool>true</bool>
          </property>
          <property name="tabPosition">
           <enum>QTabWidget::South</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_New">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="action_Project"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="menu_New"/>
    <addaction name="action_Quitter"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionEditProject"/>
    <addaction name="actionDeleteProject"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionTasks"/>
    <addaction name="actionEvents"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Project"/>
   <addaction name="actionEditProject"/>
   <addaction name="actionDeleteProject"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Projects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Calendar" name="calendar" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>180</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>180</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTasks"/>
   <addaction name="actionEvents"/>
  </widget>
  <action name="action_Project">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/std/icons/folder--plus.png</normaloff>:/std/icons/folder--plus.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Project</string>
   </property>
   <property name="iconText">
    <string>New Project</string>
   </property>
   <property name="toolTip">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Quitter">
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
  </action>
  <action name="actionDeleteProject">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/std/icons/folder--minus.png</normaloff>:/std/icons/folder--minus.png</iconset>
   </property>
   <property name="text">
    <string>Delete Project</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionEditProject">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/std/icons/folder--pencil.png</normaloff>:/std/icons/folder--pencil.png</iconset>
   </property>
   <property name="text">
    <string>Edit Project</string>
   </property>
   <property name="toolTip">
    <string>Edit Project</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionTasks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/std/icons/notebook.png</normaloff>:/std/icons/notebook.png</iconset>
   </property>
   <property name="text">
    <string>Tasks</string>
   </property>
   <property name="iconText">
    <string>View Tasks</string>
   </property>
  </action>
  <action name="actionEvents">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/std/icons/calendar.png</normaloff>:/std/icons/calendar.png</iconset>
   </property>
   <property name="text">
    <string>Events</string>
   </property>
   <property name="iconText">
    <string>View Events</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/std/icons/question.png</normaloff>:/std/icons/question.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Calendar</class>
   <extends>QWidget</extends>
   <header>widgets/calendar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
